packages/form-plugin/src/directive.ts

```diff
@@ -20,7 +20,15 @@ export class FormDirective implements OnInit, OnDestroy {
   debounce = 100;
 
   @Input('ngxsFormClearOnDestroy')
-  clearDestroy = false;
+  set clearDestroy(val: boolean) {
+    this._clearDestroy = val !== null && `${val}` !== 'false';
```

Comment:
Potentially could be:
```suggestion
    this._clearDestroy = val && `${val}` !== 'false';
```

Reply:
> Potentially could be:

this implementation is really focused on behavior change like just passing **ngxsFormClearOnDestroy** instead of doing this **ngxsFormClearOnDestroy="true"**. in your case if I pass ngxsFormClearOnDestroy it teated as false.

Reply:
null=>false (handle this condition)
undefined =>true
'false' => false
'true' => true
'' => true

Reply:
No way! I thought that an attribute with the value omitted in the HTML was given the value of the name of that attribute.
Just tested it out and you are right. You learn something new every day!

Reply:
Would we want to leave it as `value != null` except `!== null`? Since it will not work if I pass `undefined` there which is super easy when you have a property declaration in the component but it doesn't have assigned value:
```ts
@Component({
  template: '<form ngxsForm [clearOnDestroy]="clearOnDestroy">...'
})
export class CMP {
  clearOnDestroy?: unknown;
}
```
```ts
undefined !== null && `${undefined}` !== 'false' // => true
```